modules/oxide/src/less/theme/content/selection/selection.less

```diff
@@ -112,7 +112,7 @@
 
 // Table border resizing handles
 .ephox-snooker-resizer-bar {
-  background-color: @object-block-selected-color;
+  background-color: @resize-handle-color;
```

Comment:
Do we need a changelog entry for this new Oxide variable?

Reply:
The variable isn't new, it's declared in `resize-handle.less`